From 22265d3e57f0c0c4afee73b1b4efde25441577dc Mon Sep 17 00:00:00 2001
From: Paolo Bonzini <pbonzini@redhat.com>
Date: Thu, 22 Mar 2012 17:32:49 -0300
Subject: [RHEL6 qemu-kvm PATCH] fix virtio-scsi build after streaming patches

RH-Author: Paolo Bonzini <pbonzini@redhat.com>
Message-id: <1332437569-2590-1-git-send-email-pbonzini@redhat.com>
Patchwork-id: 38917
O-Subject: [RHEL 6.3 qemu-kvm PATCH] fix virtio-scsi build after streaming patches
Bugzilla: 582475
RH-Acked-by: Michal Novotny <minovotn@redhat.com>
RH-Acked-by: Eduardo Habkost <ehabkost@redhat.com>
RH-Acked-by: Jeffrey Cody <jcody@redhat.com>

Bugzilla: 582475

Brew build: http://brewweb.devel.redhat.com/brew/taskinfo?taskID=4181499

Commit 0807c76e7c948a3f5be99329a3080646337f3994 removed an include of
monitor.h from blockdev.h.  monitor.h includes (indirectly) qerror.h,
and hw/qdev.h includes blockdev.h, so virtio-scsi.c does not include
anymore qerror.h and the build fails.

Upstream, QOM started including monitor.h via qapi/qapi-visit-core.h.
Do the same here.

Please review and ack.
---
 hw/qdev.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 hw/qdev.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/hw/qdev.h b/hw/qdev.h
index b295202..9b87bd8 100644
--- a/hw/qdev.h
+++ b/hw/qdev.h
@@ -6,6 +6,7 @@
 #include "qemu-queue.h"
 #include "qemu-char.h"
 #include "qemu-option.h"
+#include "qapi/qapi-visit-core.h"
 
 typedef struct Property Property;
 
-- 
1.7.3.2

